### PR TITLE
Inital environment and seed setup (copied from previous repo)

### PR DESCRIPTION
**Pull request by brandon-win**

### CODE REVIEW

File: database/seed.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  for (let i = 0; i < numOfPics; i++) {
+    picArray.push(
+      {
+        imgId: i,
+        imgUrl: faker.image.nightlife(),
+        imgName: faker.internet.userName(),
+        imgDescription: faker.name.jobDescriptor(),
+      },
+    );
+  }
```
Comment (jx114) [inline database/seed.js:20]: Test

File: database/seed.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+/* eslint-disable no-plusplus */
+/* eslint-disable no-console */
+const faker = require('faker');
+const db = require('./index.js');
+
+// 100 different lists of images
+
+const getPictures = () => {
+  const numOfPics = Math.random() * (18 - 12) + 12;
```
Comment (jx114) [inline database/seed.js]: Should floor this to make whole ints

File: server/index.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+const express = require('express');
+
+const app = express();
+const port = 3000;
+
+app.get('/', (req, res) => {
+  res.send('Hello World!');
+});
+
+app.get('api/img_carousel');
```
Comment (jx114) [inline server/index.js]: forget request and response callback )^:<

**Review by jx114 — CHANGES_REQUESTED** [review]
Look at comments and make some changes my brother

File: database/seed.js (hunk 2 above)
Comment (brandon-win) [reply database/seed.js]: Will do!

File: server/index.js (hunk 3 above)
Comment (brandon-win) [reply server/index.js]: I'll get that done boss!

Comment (brandon-win) [reply server/index.js]: resolved!

File: database/seed.js [hunk 4]
```diff
@@ -6,7 +6,7 @@ const db = require('./index.js');
 // 100 different lists of images
 
 const getPictures = () => {
-  const numOfPics = Math.random() * (18 - 12) + 12;
+  const numOfPics = Math.round(Math.random() * (18 - 12) + 12);
```
Comment (jx114) [inline database/seed.js:9]: Good Job!

File: server/index.js [hunk 5]
```diff
@@ -8,7 +8,9 @@ app.get('/', (req, res) => {
   res.send('Hello World!');
 });
 
-app.get('api/img_carousel');
+app.get('api/img_carousel', (req, res) => {
+
```
Comment (jx114) [inline server/index.js:12]: Rmb to fill this out!

**Review by jx114 — APPROVED** [review]
Look at comments but good pull request!

**Pull request merged**